src/routes/concepts/understanding-jsx.mdx

```diff
@@ -108,14 +108,15 @@ They connect the component with the data it requires, for seamless data flows an
 
 #### Core concepts
 
-- **Static vs. dynamic attributes**:
-  In Solid's JSX, static attributes are integrated directly into the HTML template.
-  Dynamic attributes, however, are not and instead rely on state and other reactive data sources.
-  This allows the content or styling of an element to change in response to interactions or other changes within an application.
-
-- **Dynamic data transfer**:
-  Props are important when using dynamic data.
-  They let components react in real-time to changes, resulting in a responsive application.
+- **Static props**:
+  In Solid's JSX, static props are integrated directly into the HTML template as attributes. The template is cloned and used as is.
+
+- **Dynamic props**:
+  Dynamic props rely on state, allowing the content or properties to be dynamic, such changing the styling of an element  in response to interactions within an application. Mostly expressed in the form of signals `value={value()}`.
+
+- **Data transfer**:
+  Props are also used to fill components with data that comes from resources, like [`createResource`](/reference/basic-reactivity/create-resource) calls.
+  This results in components that react in real-time to data changes.
```

Comment:
```suggestion
- **Static props**:
  In Solid's JSX, static props are integrated directly into the HTML by cloning the template and using them as attributes.

 - **Dynamic props**:
 Dynamic props rely on state, allowing the content or properties to be dynamic.
 An example is changing the style of an element in response to interactions within an application. 
 This can be expressed in the form of signals (`value={value()}`).

- **Data transfer**:
  Props are also used to fill components with data that comes from resources, like [`createResource`](/reference/basic-reactivity/create-resource) calls.
  This results in components that react in real-time to data changes.
```